server/fishtest/userdb.py

```diff
@@ -35,14 +37,26 @@ def clear_cache(self):
         with self.user_lock:
             self.cache.clear()
 
-    def hash_password(self, password):
+    @lru_cache(maxsize=128)
+    def hash_password(self, password,
+        time_cost: int = 3, memory_cost: int = 65536, parallelism: int = 4,
```

Comment:
I am confused. The parameters to hash_pashword are not passes on it appears.

Reply:
I'm using the current default parameters explicitly to ensure future consistency in case the defaults are changed by the package maintainers